.github/workflows/tox-test.yml

```diff
@@ -100,6 +100,64 @@ jobs:
         uses: pypa/gh-action-pip-audit@v1.1.0
         with:
           inputs: requirements.txt requirements-test.txt
+      - name: Cache OWASP
+        uses: actions/cache@v3
```

Comment:
Could we try to use v4?

There is a warning
```
The following actions use a deprecated Node.js version and will be forced to run on node20: actions/checkout@v2, actions/setup-python@v2, actions/cache@v3. For more info: https://github.blog/changelog/2024-03-07-github-actions-all-actions-will-run-on-node20-instead-of-node16-by-default/
```
*  https://github.com/release-engineering/cloudpub/actions/runs/10515093079?pr=38

Reply:
Changed to v4 now, thanks for letting me know about the deprecation!